test/ConfigResourceTest.php

```diff
@@ -402,4 +402,80 @@ public function testDeleteNonexistentKeyShouldDoNothing()
         $test = include $this->file;
         $this->assertEquals($expected, $test);
     }
+
+    public function testDataIsSorted()
```

Comment:
Rename this to `testDataIsSortedByKey()`.

Reply:
Done :+1: